demo/README.md

```diff
@@ -37,4 +38,5 @@ This module is setup to run Jest tests, including using Enzyme and Jest Snapshot
 Jest snapshot in the HelloWorld module.  Jest tests can be run using "npm run test" or they can be run directly in IntelliJ.
 
     
-[LabKey Gradle build]: https://www.labkey.org/Documentation/wiki-page.view?name=gradleBuild    
\ No newline at end of file
+[LabKey Gradle build]: https://www.labkey.org/Documentation/wiki-page.view?name=gradleBuild   
+[Node.js Build Dependency]: https://www.labkey.org/Documentation/wiki-page.view?name=nodejs 
```

Comment:
Interesting.  I didn't know you could create links with this syntax.

Reply:
> Interesting. I didn't know you could create links with this syntax.

I didn't either, but I like it. I stole it from what Marty had already setup for the other link on this page.